server/utils/ImageUpload.ts

```diff
@@ -0,0 +1,38 @@
+import { BlobServiceClient, ContainerClient } from "@azure/storage-blob";
+
+const sasToken =
+  process.env.storagesastoken ||
+  "sv=2019-12-12&ss=b&srt=sco&sp=rwdlacx&se=2021-02-09T08:03:52Z&st=2021-02-09T00:03:52Z&spr=https&sig=huSsjMTIgztooX2xhIfFsXDxAeiLINye1%2FCzyu3%2Fnh4%3D"; // Fill string with your SAS token
```

Comment:
This SAS token will need to be put into bitwarden or an equivalent. I can regen a new one that doesn't expire so just let me know when you need that done @Sukhmai

Reply:
Actually we probably need to switch all the credentials out for the bog account (if there is one for azure).